README.md

```diff
@@ -383,6 +383,45 @@ If you have the AWS credentials configured locally in the `~/.aws` file, you can
     Replace `<container_id>` with the ID of the running container, which can be obtained by running `docker ps`.
 
 
+## Use the Docker Image from the GitHub Container Registry
+
+This repository provides a Docker image hosted in the GitHub Container Registry (GHCR) that can be used to run the application easily. Below are the steps to pull and run the Docker image.
+
+1. Pull the Docker Image
+
+```bash
+docker pull ghcr.io/swisstopo/swissgeol-boreholes-dataextraction-api:edge
+```
+
+2. a. Run the docker image from the Terminal
+
+```bash
+docker run -d --name swissgeol-boreholes-dataextraction-api -e AWS_ACCESS_KEY_ID=XXX -e AWS_SECRET_ACCESS_KEY=YYY -e AWS_ENDPOINT=ZZZ -p 8080:8080 ghcr.io/swisstopo/swissgeol-boreholes-dataextraction-api:TAG
```

Comment:
JFYI: The `AWS_ENDPOINT` environment variable is not used yet when generating the S3 URL [here](https://github.com/swisstopo/swissgeol-boreholes-dataextraction/blob/2771506c69bcfcc159209fa1cd8b60b0a0706036/src/app/api/v1/endpoints/create_pngs.py#L52).

Reply:
Thank you for raising this issue. There is an issue about that that is about to get merged, so the `AWS_ENDPOINT` environment variable will be used in this position. As this ticket will be merged before this one, I chose to already make the code compatible.